app_dart/test/foundation/utils_test.dart

```diff
@@ -231,6 +231,97 @@ void main() {
         expect(result, targets);
       });
 
+      test('returns builders when run_if matches files with ** that contain digits', () async {
+        targets = <Target>[
+          generateTarget(
+            1,
+            runIf: <String>[
+              'dev/**',
+              'packages/flutter/**',
+              'packages/flutter_driver/**',
+              'packages/integration_test/**',
+              'packages/flutter_localizations/**',
+              'packages/fuchsia_remote_debug_protocol/**',
+              'packages/flutter_test/**',
+              'packages/flutter_goldens/**',
+              'packages/flutter_tools/**',
+              'bin/**',
+              '.ci.yaml'
+            ],
+          ),
+        ];
+        files = <String>[
```

Comment:
ditto about making this only available to this test's scope, and throughout the other tests